[CC 1.0.0]Updating QSG [4.0.0] (#5121)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide-docker-with-apim.md

!!! info
    Note that the Docker Compose script deploys WSO2 API Manager with basic configurations. In order to deploy WSO2 API Manager in production grade, use the Docker setup artifacts from [APIM page](https://wso2.com/api-management/). The Docker Compose scripts are provided only for the purpose of trying it out.

> ### Step 3 - Create and publish an API from API Manager

Reviewer: "### Step 3 - Create and publish an API via API Manager"

1. Navigate to the Publisher Portal.

>     [https://apim:9443/publisher/](https://apim:9443/publisher/)

Reviewer: is this URL correct?

Author: This is correct.

    !!! tip
        Optionally, to test this service, copy the service URL [http://run.mocky.io/v2/5185415ba171ea3a00704eed](http://run.mocky.io/v2/5185415ba171ea3a00704eed) and navigate to it on a new browser. You should see the following JSON message.
            
         `{"hello": "world"}`

4. Select **REST API** from the home screen and then click **Start From Scratch**.
   
    [![Design a new REST API]({{base_path}}/assets/img/get_started/design-new-rest-api.png)]({{base_path}}/assets/img/get_started/design-new-rest-api.png)

5. Enter the API details.
     
     [![Create an API]({{base_path}}/assets/img/get_started/api-create.png){: style="width:60%"}]({{base_path}}/assets/img/get_started/api-create.png)

6. Click **Create & Publish**.

     This will publish your first API on the Developer Portal as well as deploy it on Choreo Connect. You now have an OAuth 2.0 secured REST API that is ready to be consumed.

> ### Step 4 - Subscribing to API and Get a Token

Reviewer 2: ```suggestion
### Step 4 - Subscribe to the API and Generate a Token
```

1. Navigate to the Developer Portal and and click an API (e.g., `HelloWorld`).

>     [https://apim:9443/devportal/](https://apim:9443/devportal/)

Reviewer: Is this URL correct?

Author: Correct.

> 2. Subscribe to the API (e.g., `HelloWorld` 1.0.0) using an application and an available throttling policy.

Reviewer: "2. Subscribe to the API (e.g., `HelloWorld` 1.0.0) using an application and an available Rate Limiting Policy."

    **JWT tokens**:
    As the application is self-contained (JWT), **copy the generated access token** before proceeding to the next step.

### Step 5 - Invoke the API via Choreo Connect

> Follow the instructions below to invoke the previously created API with the generated keys.

Reviewer 2: ```suggestion
Follow the instructions below to invoke the previously created API with the generated token.
```

1. Click **Try Out** on the left menu bar.

     The resources of the API will be listed.

2. Paste the access token that you previously copied in the **Access Token** field.

>     [![Paste the access token]({{base_path}}/assets/img/deploy/mgw/invoke-api.png)]({{base_path}}/assets/img/deploy/mgw/invoke-api.png)

Reviewer: Crop the bottom of this image in order to remove the footer, which appears in black.

> 3. **If this is the first time you are using the API test console** from your browser,  open a new tab and navigate to the [https://localhost:9095/](https://localhost:9095/) URL.

Reviewer 3: ```suggestion
3. **If this is the first time you are using the API test console** from your browser, open a new tab and navigate to the [https://localhost:9095/](https://localhost:9095/) URL.
```

>      This will prompt your browser to accept the certificate used by the Choreo Connect. This is required because by default the Choreo Connect uses a self-signed certificate that is not trusted by web browsers.

Reviewer 3: ```suggestion
     This will prompt your browser to accept the certificate used by Choreo Connect. This is required because, by default, Choreo Connect uses a self-signed certificate that is not trusted by web browsers.
```

    
    !!! note

>         This certificate that is used by the Choreo Connect is replaced when deploying the system in production.

Reviewer 3: ```suggestion
        This certificate that is used by Choreo Connect is replaced when deploying the system in production.
```

> 4. Click on the `GET` resource of the API to expand the resource and Click **Try It Out**.

Reviewer 3: ```suggestion
4. Click on the `GET` resource of the API to expand the resource and click **Try It Out**.
```

   
     [![GET resource]({{base_path}}/assets/img/deploy/mgw/expanded-get-resource.png)]({{base_path}}/assets/img/deploy/mgw/expanded-get-resource.png)

5. Click **Execute**.

     [![GET resource]({{base_path}}/assets/img/deploy/mgw/try-api.png)]({{base_path}}/assets/img/deploy/mgw/try-api.png)

     You should see the `{"hello" : "world"}` response from the API. 

     [![Successful response]({{base_path}}/assets/img/deploy/mgw/try-it-success.png)]({{base_path}}/assets/img/deploy/mgw/try-it-success.png)

> __Congratulations!__ You have successfully created your first API, subscribed to it through an OAuth 2.0 application, obtained an access token for testing, and invoked your API with the Choreo Connect.

Reviewer 3: ```suggestion
__Congratulations!__ You have successfully created your first API, subscribed to it through an OAuth 2.0 application, obtained an access token for testing, and invoked your API with Choreo Connect.
```
